src/modules/RA/DataLayer/components/CustomFormTab.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
This file should be on more common place. We can imagine this component can be useful in another part.

Reply:
Alright

Reply:
As it is a custom react-admin component, It has been moved to `src/components/react-admin`.